publication-file/src/main/java/no/unit/nva/publication/file/upload/FileService.java

```diff
@@ -89,6 +88,26 @@ public UploadedFile completeMultipartUpload(SortableIdentifier resourceIdentifie
         return file;
     }
 
+    public void deleteFile(UUID fileIdentifier, SortableIdentifier resourceIdentifier, UserInstance userInstance)
+        throws ForbiddenException, NotFoundException {
+        var resource = fetchResource(resourceIdentifier);
+
+        validateDeletePermissions(userInstance, resource);
```

Comment:
You're doing it here, which is possibly an artefact of our weird logic for managing permissions